src/server/address_manager_store.py

```diff
@@ -0,0 +1,214 @@
+from src.server.address_manager import (
+    AddressManager,
+    ExtendedPeerInfo,
+    NEW_BUCKET_COUNT,
+    BUCKET_SIZE,
+    NEW_BUCKETS_PER_ADDRESS,
+)
+
+import logging
+import aiosqlite
+
+log = logging.getLogger(__name__)
+
+
+class AddressManagerStore:
```

Comment:
We might want to put this class in the full node dir (only the full node uses it right?)